terraform/modules/external_domain_broker_govcloud/resources.tf

```diff
@@ -0,0 +1,26 @@
+# These resources are for https://github.com/cloud-gov/external-domain-broker
+# Note that the broker needs both commercial and govcloud users because it operates
+# on both commercial (cloudfront, route53) and govcloud (alb) resources
+
+data "template_file" "policy" {
+  template = "${file("${path.module}/policy.json")}"
+
+  vars {
+    account_id        = "${var.account_id}"
+    stack             = "${var.stack_description}"
+  }
+}
+
+resource "aws_iam_user" "iam_user" {
+  name = "external-domain-broker-${var.stack_description}"
```

Comment:
worth noting that this user has the same name as the user in commercial. This should not create any direct conflict, but maybe it's confusing?

Reply:
I would say that adding `-govcloud` would make it easier for future selfs. Everything else looks good

Reply:
added `-gov`